responsibleai/tests/test_model_analysis.py

```diff
@@ -51,6 +52,7 @@ def test_model_analysis_iris(self, manager_type):
         X_train[LABELS] = y_train
         X_test[LABELS] = y_test
         manager_args = {
+            ManagerParams.TREATMENT_FEATURES: [feature_names[0]],
```

Comment:
Personally I might avoid `feature_names[0]` in case the order of feature names changes, but it should be fine

Reply:
It should be fine as the causal test is only for the assert condition.